builtin/settingtypes.txt

```diff
@@ -619,20 +770,21 @@ profiler_print_interval (Profiling print interval) int 0
 #    memory consumption (4096=100MB, as a rule of thumb).
 max_clearobjects_extra_loaded_blocks (Max. clearobjects extra blocks) int 4096
 
-#    How much the server will wait before unloading unused MapBlocks.
+#    How much the server will wait before unloading unused mapblocks.
 #    Higher value is smoother, but will use more RAM.
 server_unload_unused_data_timeout (Unload unused server data) int 29
 
 #    Maximum number of statically stored objects in a block.
 max_objects_per_block (Maxmimum objects per block) int 49
 
-#    http://www.sqlite.org/pragma.html#pragma_synchronous only numeric values: 0 1 2
-sqlite_synchronous (Synchronous SQLite) int 2 0 2
+#    See http://www.sqlite.org/pragma.html#pragma_synchronous
+sqlite_synchronous (Synchronous SQLite) enum 2 0,1,2
 
 #    Length of a server tick and the interval at which objects are generally updated over network.
 dedicated_server_step (Dedicated server step) float 0.1
 
-#    Can be set to true to disable shutting down on invalid world data.
+#    Can be enabled to disable shutting down on invalid world data.
```

Comment:
Perhaps it would be easier to understand if phrased "If enabled, invalid world data won't cause the server to shut down"?